lesson_2/Lesson_2.4.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>single number in a continuous <strong>distribution</strong> is <strong>0</strong></p><p> </p><blockquote>Note though, we don't need to know the probability of any one observed value, what we need is the relative values which the probability density function gives us</blockquote><p>Make sure to give a bit more details about that. That's not very clear currently. In particular, relative to what?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>